wire-library/wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -1495,11 +1496,13 @@ class KotlinGenerator private constructor(
         .build()
   }
 
-  private fun Field.redact(fieldName: String): CodeBlock? {
+  private fun Field.redact(fieldName: String, isOneOf: Boolean): CodeBlock? {
     if (isRedacted) {
       return when {
         isRepeated -> CodeBlock.of("emptyList()")
         isMap -> CodeBlock.of("emptyMap()")
+        isOptional || isOneOf -> CodeBlock.of("null")
```

Comment:
Is there an existing test case for one ofs?

Reply:
Also could we look at `Field.EncodeMode.NULL_IF_ABSENT` instead?

Reply:
Hrmm not for this use case. I've updated the included test to cover redacted one ofs and switched over to `Field.EncodeMode.NULL_IF_ABSENT` as well, thanks.